apps/finance/app/src/components/NewTransfer/Deposit.js

```diff
@@ -318,18 +318,14 @@ class Deposit extends React.Component {
         {errorMessage && <ValidationError message={errorMessage} />}
 
         <VSpace size={4} />
-        <Info title="Depositing funds to your organization">
+        <Info>
           {isMainnet && (
-            <React.Fragment>
-              <p>
-                Remember, Mainnet organizations use real (not test) funds.{' '}
-                <StyledSafeLink href={MAINNET_RISKS_BLOG_POST} target="_blank">
-                  Learn more
-                </StyledSafeLink>{' '}
-                about the risks and what's been done to mitigate them here.
-              </p>
-              <VSpace size={1} />
-            </React.Fragment>
+            <p>
+              Remember, Mainnet organizations use <strong>real tokens</strong>.
```

Comment:
cc @dizzypaty I would still suggest "funds" as the terminology, since ETH is not _really_ a token.